include/envoy/common/platform.h

```diff
@@ -189,6 +189,18 @@ struct msghdr {
 #define IP6T_SO_ORIGINAL_DST 80
 #endif
 
+#ifndef SOL_UDP
+#define SOL_UDP 17
+#endif
```

Comment:
You probably don't need this, this is in libc for 20+ years.

Reply:
I had initially skipped this definition, but some macOS CIs failed expecting `SOL_UDP` to be defined. 
Hence, I put it in here! :)